napari/layers/utils/_tests/test_stack_utils.py

```diff
@@ -0,0 +1,124 @@
+from napari.layers.image.image import Image
+from napari.layers.utils.stack_utils import stack_to_images, images_to_stack
+import numpy as np
+
+
+def test_stack_to_images_basic():
+
+    '''
+    Test 2 channels, no colormap
+    '''
```

Comment:
```suggestion
    """Test 2 channels, no colormap."""
```
([PEP 257](https://www.python.org/dev/peps/pep-0257/))

Also, if possible, I think it's nice to say what expected behavior you're testing.  like
```python
"""Test that a multichannel stack is split into single-channel layers."""
```